pydocxs3upload/image_upload.py

```diff
@@ -88,10 +109,11 @@ def upload(self, img_data, filename, image_format=None):
 
         data = self.signed_data
 
-        data['Content-Type'] = 'image/%s' % image_format
-
         url = self.s3_url
 
+        if 'storage.googleapis.com' not in url:
```

Comment:
When using GCS, this is already present in the signed_data. Adding it again causes the request to not match the policy.